fixed no schema matches error (#895)
strimzi/strimzi-kafka-bridge: CHANGELOG.md

## 0.29.0

* Dependency updates (Vert.x 4.5.7, Netty 4.1.108.Final)

> * Added support for records key and value to be JSON array when using the JSON embedded format.

Reviewer: ```suggestion
* Added support for records key and value to be JSON array when using the `json` embedded format.
```

> * Fixed No schema matches error when using the `json` embedded format and the record key or value is an array.

Reviewer: I would reword and changing in terms of added support for key and value to be JSON array and not fixed bug imho.

Author: Maybe something like `Added support for key and value to be JSON array. Works with `json` embedded format.` would better?

Reviewer: Maybe a mix of that like "Added support for records key and value to be JSON array when using the `json` embedded format".

Author: Nice one!

> * Fixed No schema matches error when using the `json` embedded format and the record key or value is an array.

Author: ```suggestion
* Added support for records key and value to be JSON array when using the json embedded format.
```
